tests/gha-run-k8s-common.sh

```diff
@@ -264,3 +264,54 @@ function deploy_k8s() {
 
 	echo "::endgroup::"
 }
+
+function install_snapshotter(){
```

Comment:
nit: Add space before function block for consistency
```suggestion
function install_snapshotter() {
```

Reply:
Fixed.